crates/signer/src/signer.rs

```diff
@@ -0,0 +1,116 @@
+use alloy_signer_local::PrivateKeySigner;
+use eth_keystore::decrypt_key;
+use std::path::Path;
+use thiserror::Error;
+
+#[derive(Debug)]
+/// Represents the input params to create a signer
```

Comment:
nit can you put the docs above the derive? here and else where

Reply:
Done!